antsibull/config.py

```diff
@@ -146,6 +148,20 @@ def convert_to_none(cls, value):
             value = None
         return value
 
+    @p.validator('breadcrumbs', 'indexes', pre=True)
+    def convert_to_bool(cls, value):
+        if isinstance(value, str):
```

Comment:
I think we should move this to a common utility function so that both versions of this code won't get out of sync so easily.

Reply:
Fixed in 7d717282a48792c671de1963fb204e868c613ec9

I ended up restructuring so that all schemas are in the `schemas/` directory.  The documentation schemas moved to `schemas/docs`.  config and context get their own python modules in schemas.  The common code is all validator  related so it's in: https://github.com/ansible-community/antsibull/pull/292/commits/7d717282a48792c671de1963fb204e868c613ec9#diff-02cd3e7ff7dcd7b35c8080c275011994c07d608c3e9be75336848184f1c498b1

I did not audit the docs schemas to look for things that should move into common code at this time so it's probable that there's code in there that should be consolidated into the common modules later.